code/style.css

```diff
@@ -1,4 +1,185 @@
-html {
-  background: #0025ff;
-  color: #fff;
+/* BODY */
+
+body {
+  background: rgb(231, 103, 89);
+  color: rgb(76, 223, 76);
+  font-family: 'Potta One';
+  margin: 0;
+}
+
+/* HEADER */
+
+header {
+  background: white;
+  width: 100%;
+  height: 100px;
+  display: flex; 
+  align-items: center;
+  padding: 0 16px;
+}
```

Comment:
Having both 100% width and some padding will make the content too wide. Consider removing the width and let the header be as big as it's allowed. The blue mark is what fits in the HTML:

<img width="407" alt="Skärmavbild 2021-02-02 kl  11 27 14" src="https://user-images.githubusercontent.com/51529201/106587191-9adb1880-6549-11eb-8d59-1215531a62ab.png">